modules/ossm-member-roll-create.adoc

```diff
@@ -22,11 +22,19 @@ Follow this procedure to add one or more projects to the {ProductShortName} memb
 
 .Prerequisites
 * An installed, verified {ProductName} Operator.
-* Location of the installed `ServiceMeshControlPlane`.
-* List of projects to add to the service mesh.
```

Comment:
* List of existing projects that you want to have the service mesh monitor.

Reply:
The changes you requested have been updated, @JStickler and I marked them resolved. This isn't ready to merge until 1.0.10 is released.